keyboards/sergiopoverony/creator_pro/config.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2020-2021 SergioPoverony
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "config_common.h"
+
+/* vid pid  */
+#define VENDOR_ID       0x8B1E 
+#define PRODUCT_ID      0x0C00
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    SergioPoverony
+#define PRODUCT         Creator Pro
+
+
+/* Optimise size swith off unused functions*/
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+#define NO_ACTION_TAPPING
+#define NO_ACTION_ONESHOT	
```

Comment:
Not a fan of these being defined, as they disable a lot of functionality

Reply:
fixed.